main.cpp

```diff
@@ -73,45 +92,75 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
- */
+ 1)
 
+ */
+ int adder(int a, int b) {
+     ignoreUnused(a,b);
+     return {};
+ }

```

Comment:
please follow the coding style for this course
Curly braces go on their own line
- [ ] revise